components/CTA/CTA.tsx

```diff
@@ -0,0 +1,25 @@
+import type { NextPage } from "next";
+import Link from "next/Link";
+import Chevron from "components/Chevron/Chevron";
+import styles from "styles/CTA.module.scss";
```

Comment:
I’m surprised we wouldn’t colocate the styles and the React component side-by-side in the same folder? Feels like it makes it easier to move around the code, if the styles are per component.